dymos/transcriptions/explicit_shooting/ode_integration_comp.py

```diff
@@ -664,8 +665,8 @@ def _f_primal(self, t, x, theta):
         ----------
         t : float
             The current value of the integration variable.
-        y : np.array
-            The augmented state vector.
+        x : np.array
```

Comment:
Some part of this docstring doesn't jive; right after the function definition, it says "where y is the augmented state vector" but now there's no y? And dtheta_dz isn't to be seen either? Maybe just updating the comments is all that's needed